crates/frontend/src/components/toast/mod.rs

```diff
@@ -0,0 +1,72 @@
+use leptos::*;
+
+#[derive(PartialEq, Clone, Debug, strum_macros::Display)]
+#[strum(serialize_all = "lowercase")]
+pub enum AlertType {
+    Info,
+    Success,
+    Warning,
+    Error,
+}
+
+#[derive(Clone, Debug)]
+pub struct Alert {
+    pub id: u64,
```

Comment:
is ID required?

Reply:
I need it for removing the correct alert after the timeout.